test/checks/missing_enable_comment_test.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+require "test_helper"
+require "minitest/focus"
+
+class MissingEnableCommentTest < Minitest::Test
+  def test_always_enabled_by_default
+    assert(ThemeCheck::MissingEnableComment.new.always_enabled?)
+  end
+
+  def test_no_default_noops
+    offenses = analyze_theme(
+      ThemeCheck::MissingEnableComment.new,
+      "templates/index.liquid" => <<~END,
+        {% theme-check-disable %}
```

Comment:
This is a Liquid syntax error (because the tag doesn't exist). The `SyntaxError` check would report it. Is this what you wanted to test?

Reply:
Whoops! Great catch.